ingestion/src/main/java/feast/ingestion/utils/StoreUtil.java

```diff
@@ -101,155 +78,19 @@
     VALUE_TYPE_TO_STANDARD_SQL_TYPE.put(Enum.BOOL_LIST, StandardSQLTypeName.BOOL);
   }
 
-  public static void setupStore(Store store, FeatureSet featureSet) {
+  public static FeatureSink getFeatureSink(
```

Comment:
I have mixed feelings about introducing a new term here (`FeatureSink`). Does it carry any unique meaning about a store?

Reply:
It's to distinguish stores that store features v.s. deadletter rows.